interpro7dw/alphafold.py

```diff
@@ -21,26 +38,34 @@ def export(alphafold_file: str, proteins_file: str, output: str,
     with KVStore(proteins_file) as st:
         keys = st.get_keys()
 
-    with KVStoreBuilder(output, keys=keys, tempdir=tempdir) as ash:
-        with open(alphafold_file, "rt") as fh:
-            for line in fh:
-                """
-                Columns:
-                    - UniProt accession, e.g. A8H2R3
-                    - AlphaFold DB identifier, e.g. AF-A8H2R3-F1
-                    - mean pLDDT of the prediction
-                """
-                cols = line.rstrip().split()
-                uniprot_acc = cols[0]
-                alphafold_id = cols[1]
-                score = float(cols[2])
-                ash.add(uniprot_acc, (alphafold_id, score))
-
-        if keep_fragments:
-            ash.build(apply=lambda x: x)
-        else:
-            ash.build(apply=lambda x: x if len(x) == 1 else [])
-
-        logger.info(f"temporary files: {ash.get_size() / 1024 ** 2:.0f} MB")
+    with KVStore(proteins_file) as protein:
+        with KVStoreBuilder(output, keys=keys, tempdir=tempdir) as ash:
```

Comment:
Can directly get keys from the KVStore instance:

```suggestion
    with KVStore(proteins_file) as ks:
        with KVStoreBuilder(output, keys=ks.get_keys(), tempdir=tempdir) as kb:
```